ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -67,11 +67,29 @@ <h3 class="title">
         color="primary"
         class="icon drop"
         (click)="dropIndex()"
-        *ngIf="currentObject!.isSpannerNode && currentObject!.type === 'indexName'"
+        *ngIf="
```

Comment:
The count of indexes should get modified once the index is deleted : https://drive.google.com/file/d/1yEcLrr0Ybhpa1HdgTPxkMGP373u98qJG/view?usp=share_link&resourcekey=0-1gymAPFb0NCsB4Q8ZSGEWQ